public/components/FilterBadge.vue

```diff
@@ -55,9 +55,6 @@ export default Vue.extend({
   computed: {
     filterName(): string {
       const type = getVarType(this.filter.key);
-      if (isClusterType(type)) {
```

Comment:
You can remove the function import as well

Reply:
Will do.